src/domain/LottoConfirm.js

```diff
@@ -0,0 +1,85 @@
+import {
+  LOTTO_3RD_PRIZE_WINNER,
+  LOTTO_4TH_PRIZE_WINNER,
+  LOTTO_5TH_PRIZE_WINNER,
+  LOTTO_BONUS_COUNT,
+  LOTTO_FIRST_PRIZE_WINNER,
+  LOTTO_SECOND_PRIZE_WINNER,
+} from '../constants';
+import LottoValidator from './LottoValidator';
+
+class LottoConfirm {
+  #winnigNumbers;
+  #bonusNumber;
+
+  constructor() {
+    this.#winnigNumbers = [];
+    this.#bonusNumber = 0;
+    this.validators = new LottoValidator();
+  }
```

Comment:
생성자가 큰 의미가 없는 상황처럼 보이는데, 아래처럼 변경 가능하겠네요!

```suggestion
  #winnigNumbers = [];
  #bonusNumber = 0;
  validators = new LottoValidator();
```

Reply:
기본값을 넣을때는 무조건 contructor를 사용해야되는줄 알았습니다 ㅜㅜ 이게 된다면 이게 더 좋은방법인거같습니다.